src/ray/core_worker/transport/actor_submit_queue.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2017 The Ray Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#pragma once
+
+#include <map>
+#include <utility>
+
+#include "absl/types/optional.h"
+#include "ray/common/id.h"
+#include "ray/common/task/task_spec.h"
+
+namespace ray {
+namespace core {
+
+/**
+ * IActorSubmitQueue is responsible for queuing per actor requests on the client.
+ * TODO(scv119): add more details.
+ */
+class IActorSubmitQueue {
```

Comment:
Existing interface classes are named `Interface` as the suffix. Is there any reason for using such rule?